spec/support/sdam_monitoring/standalone.yml

```diff
@@ -0,0 +1,74 @@
+description: "Monitoring a standalone connection"
+uri: "mongodb://a:27017"
+phases:
+  -
+    responses:
+      "a:27017":
+        ok: 1
+        ismaster: true
+        minWireVersion: 0
+        maxWireVersion: 4
+    outcome:
+      events:
+        -
+          topology_opening_event:
+            topologyId: "42"
+        -
+          server_opening_event:
+            topologyId: "42"
+            address: "a:27017"
+        -
+          server_description_changed_event:
+            topologyId: "42"
+            address: "a:27017"
+            previousDescription:
+              address: "127.0.0.1:27017"
```

Comment:
This address surprises me?  Where does it come from?  Why would a driver have that as the default address?  I'd expect instead that the previousDescriptions's address would be the same as the newDescription's address, but with type "Unknown" (as it is).